tombstone_gc.cc

```diff
@@ -299,7 +314,6 @@ void validate_tombstone_gc_options(const tombstone_gc_options* options, data_dic
     }
 
     const replica::keyspace& ks = db.real_database().find_keyspace(ks_name);
-    if (options->mode() == tombstone_gc_mode::repair && !needs_repair_before_gc(ks.get_replication_strategy(), real_db_ptr->get_token_metadata())) {
-        throw exceptions::configuration_exception("tombstone_gc option with mode = repair not supported for table with RF one or local replication strategy");
-    }
+    return validate_tombstone_gc_options(options, db.features(),
+            ks.get_replication_strategy(), real_db_ptr->get_token_metadata());
```

Comment:
Nit: you're returning a `void` value in a `void` function. This is allowed by C++, but it's rather confusing in non-template code.